Source/Urho3D/CSharp/Urho3DNet.csproj

```diff
@@ -60,4 +60,4 @@
       <Link>Urho3D.pdb</Link>
     </Content>
   </ItemGroup>
-</Project>
+</Project>
```

Comment:
New line at the end of file, please

Reply:
not sure why as well!!

Reply:
Visual Studio saved .csproj.

Reply:
not only that it also omit <compile *.cs/> and add all the cs files literally. 😣